### DISCUSSION TITLE
MY_SIG and MY_SIG_INFO Fields missing for ADIF Exports

### DISCUSSION THREAD

**Discussion by DJLax5** [magicbug/Cloudlog#3120] [Q&A]
Hi everyone! 

I've noticed the MY_SIG and MY_SIG_INFO fields are (no longer?) exported if I export the ADIF for my station locations? Sadly the WWFF programme requires those fields. I am sure they were exported once. Is that gone or is there an issue on my side? 
(Obviously the fields are populated under my specific station location....)

Has anyone else that issue?

Thanks and 73,
Fabian

**Comment by phl0** [comment]
I think I implemented recent ADIF standard which has own separate fields for those. See http://adif.org.uk/314/ADIF_314.htm#QSO_Field_MY_WWFF_REF. So they should read those fields instead of SIG.

**Reply by DJLax5** [reply]
Agreed, regarding what WWFF _should_ read. 
But ... aren't the fields MY_SIG and MY_SIG_INFO also part of the adif standard? https://adif.org.uk/314/ADIF_314.htm#QSO_Field_MY_SIG_INFO

**Reply by phl0** [reply]
They are. But why use those general purpose fields if there are specific ones? SIG and SIG_INFO are used by many other programs as well.

**Reply by DJLax5** [reply]
Okay, I see what you mean, I've just tried it. 

If I populate the MY_SIG field to something _other_ than WWFF, those fields are exported to the ADIF file just fine.  Kind of surprising that there is a filter in place. But whatever, I agree following the ADIF standard is what Cloudlog should do.

Thanks for the quick response!

**Reply by phl0** [reply]
Can't remember to have made a filter here. But anyway: IMHO it makes sense if they used the dedicated fields. Why else should they be there ...
